components/email_aliases/email_aliases_service_unittest.cc

```diff
@@ -259,7 +267,7 @@ class EmailAliasesServiceTimingTest : public ::testing::Test {
     const GURL verify_result_url =
         EmailAliasesService::GetAccountsServiceVerifyResultURL();
     url_loader_factory_.SetInterceptor(base::BindLambdaForTesting(
-        [&](const network::ResourceRequest& request) {
+        [&, verify_result_url](const network::ResourceRequest& request) {
```

Comment:
There was an error, `verify_result_url` was becoming invalid on the second (and other) call.